doc/contributing/hw/racl/README.md

```diff
@@ -202,147 +163,177 @@ Components that bridge fabric transactions may substitute the role in incoming t
+        allowed_wr: [ "ROT" ]
+      }
+      { name: "SOC_ROT"
+        desc: "Custom policy allowing SOC and ROT"
+        allowed_rd: [ "ROT", "SOC" ]
+        allowed_wr: [ "ROT", "SOC" ]
+      }
     ]
   }
 }
 ```
 
 ### Instance-level Policy Mapping
 
-RACL policies apply on a per-instance basis, meaning that different instances of the same IP have a different RACL policy.
-To do so, the integrator has to define a mapping of each register of an instance to one of the policies defined in `racl.hjson`.
-An example mapping can be seen below, defining the mapping for a SPI host.
-The mapping is defined as a single HJSON file per instance, and is added as a new entry in the top-level HJSON definition:
+RACL policies apply on a per-instance basis, meaning that different instances of the same IP can be protected using different RACL policies, or different RACL groups.
+To do so, the integrator defines a mapping of each register or window of an instance to one of the policies defined in `racl.hjson`.
+As a short-hand, `*` may be used as a catch-all in case a register is not explicitly named in the configration file.
+The following shows a few different examples of an instance-level policy mapping:
+
+1. This minimal configuration specifies that for the RACL controller group `Null`, all registers shall be protected using the `SOC_ROT` policy.
 ```
 {
- policy_group: "default_group"
- policy_mapping: {
-   "INTR_STATE"   : "ROT_PRIVATE"
-   "INTR_ENABLE"  : "ROT_PRIVATE"
-   "INTR_TEST"    : "ROT_PRIVATE"
-   "ALERT_TEST"   : "ROT_PRIVATE"
-   "CONTROL"      : "ROT_PRIVATE"
-   "STATUS"       : "ALL_RD_WR"
-   "CONFIGOPTS"   : "ROT_PRIVATE"
-   "CSID"         : "ROT_PRIVATE"
-   "COMMAND"      : "ROT_PRIVATE"
-   "RXDATA"       : "ROT_PRIVATE"
-   "TXDATA"       : "ROT_PRIVATE"
-   "ERROR_ENABLE" : "ROT_PRIVATE"
-   "ERROR_STATUS" : "SOC_ROT"
-   "EVENT_ENABLE" : "ROT_PRIVATE"
- }
+  Null: {
+    registers: {
+      "*": "SOC_ROT"
+    }
+  }
 }
 ```
 
-This instance mapping is added to the instantiation in the top-level HJSON as defined as follows.
+2. RACL also supports so-called _ranges_, which can be used by a RACL-enabled IP to protect whole address ranges, e.g., for protecting different SRAM address ranges.
+The `*` option does not exist for ranges. These must be explicitly defined as follows.
+The given configuration defines one range with 4KiB in size protected by the `ROT_PRIVATE` policy, and another 4 Byte range using the `ALL_RD_WR` policy.
+```
+{
+  Null: {
+    registers: {
+      "*": "SOC_ROT"
+    }
+    windows: {
+      "*": "SOC_ROT"
+    }
+    ranges: [
+        {
+            'base': "0x0000"
+            'size': "0x1000"
+            'policy': "ROT_PRIVATE"
+        }
+        {
+            'base': "0x1000"
+            'size': "0x0004"
+            'policy': "ALL_RD_WR"
+        }
+    ]
+  }
+}
+```
+
+3. Instead of `*`, individual registers and windows can be protected with different policies when named explicitly:
```

Comment:
Grammar nit: I think you probably mean "... by naming them explicitly" (because otherwise it sounds like "you can only protect a register if it has been explicitly named")